server/views/pages/overview/partials/workAndInterestsTab/_inductionLongQuestionSet.njk

```diff
@@ -55,15 +55,15 @@ questions are different.
           {% for workExperienceDetails in workAndInterests.data.workExperience.jobs %}
             <div class="govuk-summary-list__row">
               <dt class="govuk-summary-list__key">
-                {% if workExperienceDetails.type == 'Other' %}{{ workExperienceDetails.other }}{% else %}{{ workExperienceDetails.type | formatJobType }}{% endif %} experience details
+                {% if workExperienceDetails.type == 'Other' %}Other - {{ workExperienceDetails.other }}{% else %}{{ workExperienceDetails.type | formatJobType }}{% endif %} experience details
               </dt>
               <dd class="govuk-summary-list__value">
                 <p>Job role:<br> {{ workExperienceDetails.role }}</p>
                 <p>Roles and responsibilities:<br> {{ workExperienceDetails.responsibilities }}</p>
               </dd>
               <dd class="govuk-summary-list__actions govuk-!-display-none-print">
                 <a class="govuk-link" href="{{ ciagInductionUrl }}/plan/create/{{ prisonerSummary.prisonNumber }}/work-details/{% if workExperienceDetails.type == 'Other' %}{{ workExperienceDetails.other | lower }}{% else %}{{ workExperienceDetails.type | lower }}{% endif %}/update">
-                  Change<span class="govuk-visually-hidden"> {% if workExperienceDetails.type == 'Other' %}{{ workExperienceDetails.other }}{% else %}{{ workExperienceDetails.type | formatJobType }}{% endif %} experience details</span>
+                  Change<span class="govuk-visually-hidden"> {% if workExperienceDetails.type == 'Other' %}Other - {{ workExperienceDetails.other }}{% else %}{{ workExperienceDetails.type | formatJobType }}{% endif %} experience details</span>
```

Comment:
This one probably doesn't need changing does it? 🤔 
Otherwise it'll read out the phrase `Change Other - Ice Cream Man experience details` won't it ?